lib/internal/Magento/Framework/Webapi/Rest/Response/Renderer/Xml.php

```diff
@@ -111,8 +111,7 @@ protected function _formatValue($value)
             /** Without the following transformation boolean values are rendered incorrectly */
             $value = $value ? 'true' : 'false';
         }
-        $replacementMap = ['&' => '&amp;'];
-        return str_replace(array_keys($replacementMap), array_values($replacementMap), $value);
+        return $value;
```

Comment:
must be `return (string) $value` because the function can get passed floats as well (the docblock lies) -> this causes a regression with prices in the XML

Reply:
@amenk Fixed your comment.

Reply:
I can't seem to see the change in this PR ? Do you have a link or did you forget to push?

Reply:
https://github.com/magento/magento2/pull/18362/commits/0f75306061dcf6dc2942d6e165e8ba17f2ce3f6c

Reply:
A you disguised it as "added unit test" :-)

Reply:
@amenk Made an amend commit)